### PR TITLE
Introduce pkg_arch and pkg_ext attributes

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

**Review by adamlazik1 — APPROVED** [review]

File: guides/common/attributes-base.adoc [hunk 1]
```diff
@@ -111,3 +111,5 @@
 :SmartProxyServerTitle: Smart{nbsp}Proxy{nbsp}Server
 :Team: Foreman developers
 :sectanchors:
+:pkg_arch: noarch
+:pkg_type: rpm
```
Comment (ekohl) [inline guides/common/attributes-base.adoc:115]: Isn't this rather the package (file) extension?

**Review by ekohl — CHANGES_REQUESTED** [review]
These properties are only used in examples. I'd even think about using `example_pkg_*` names.

File: guides/common/attributes-base.adoc [hunk 2]
```diff
@@ -111,3 +111,5 @@
 :SmartProxyServerTitle: Smart{nbsp}Proxy{nbsp}Server
 :Team: Foreman developers
 :sectanchors:
+:pkg_arch: noarch
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/attributes-base.adoc:114]: My PR tries to not change anything. Going with a different arch for the example package is OK for me, but unrelated to what I'm trying to achive.

[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/attributes-base.adoc:114]: Could you elaborate on what you're trying to achieve? The commit and PR are light on details.

File: guides/common/attributes-base.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/attributes-base.adoc]: Renamed.

**Comment by maximiliankolb** [comment]
Is this OK for you @ekohl ?

**Comment by maximiliankolb** [comment]
Are there any other comments or suggestions? If not, I'd like to merge this by the end of tomorrow.

**Comment by adamlazik1** [comment]
I am fine with the change.

**Review by Lennonka — APPROVED** [review]
Makes sense to me.

**Review by ekohl — COMMENTED** [review]
I can agree on the content, but would it make sense to clean up the commits and PR title to match the content?

**Comment by maximiliankolb** [comment]
I have rebased to master and squashed my commits.

**Pull request merged**